mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/IDefaultNodeAdapter.kt

```diff
@@ -96,4 +97,8 @@ interface IDefaultNodeAdapter : IDeprecatedNodeDefaults {
         val link = child.getContainmentLink() ?: error("ContainmentLink not found for node $child")
         throw UnsupportedOperationException("Cannot remove child in link $link of concept $concept.")
     }
+
+    override fun replaceNode(concept: ConceptReference): INode {
+        throw UnsupportedOperationException("Node replacement is not possible for node: $this")
```

Comment:
nit: Any reason to override the default `replaceNode` implementation here, that already throws an `UnsupportedOperationException`?

Reply:
override doesn't exist anymore